encoding/kzg/prover/parametrized_prover.go

```diff
@@ -67,30 +73,56 @@ func (g *ParametrizedProver) Encode(inputFr []fr.Element) (*bn254.G1Affine, *bn2
 
 	encodeStart := time.Now()
 
-	rsChan := make(chan RsEncodeResult, 1)
-	lengthCommitmentChan := make(chan LengthCommitmentResult, 1)
-	lengthProofChan := make(chan LengthProofResult, 1)
-	commitmentChan := make(chan CommitmentResult, 1)
-	proofChan := make(chan ProofsResult, 1)
+	commitmentsChan := make(chan commitmentsResult, 1)
```

Comment:
Just making sure, the changes here are just refactors to allow us to expose GetFrames and GetCommitments. None of the functionality changes in any major way?

Reply:
That's right!